silnlp/common/paratext.py

```diff
@@ -203,11 +203,11 @@ def extract_major_terms_per_language(iso: str) -> None:
 
     terms_glosses_path = get_terms_glosses_path(list_name="Major", iso=iso)
 
-    with open(terms_glosses_path, "w", encoding="utf-8", newline="\n") as terms_glosses_file:
+    with terms_glosses_path.open("w", encoding="utf-8", newline="\n") as terms_glosses_file:
         # import major metadata to line up terms to it
-        major_metadata = open(
-            SIL_NLP_ENV.assets_dir / "Major-metadata.txt", "r", encoding="utf-8", newline="\n"
-        ).readlines()
+        major_metadata = (
```

Comment:
I know this code was already not using a `with` statement, but can you fix it to use one?

Reply:
Done.